lib/processors/manifestCreator.js

```diff
@@ -399,9 +399,9 @@ async function createManifest(libraryResource, libBundle, descriptorVersion, _in
 				let i18nElement = findChild(libraryAppData, "i18n");
 				if ( i18nElement ) {
 					let i18n = i18nElement._;
-					if ( "false".equals(i18n) ) {
```

Comment:
Peter suggested to polyfill `Object.prototype.equals` to help :older_man: with :coffee: background.

Reply:
Hahaha 😄